lib/testutils/httpmultibin.go

```diff
@@ -164,6 +164,29 @@ func getEncodedHandler(t testing.TB, compressionType httpext.CompressionType) ht
 	})
 }
 
+func getZstdBrHandler(t testing.TB) http.Handler {
```

Comment:
Very minor, but what do you think about merging this helper function into `getEncodedHandler()`, considering they're so similar?

The signature might look like `getEncodedHandler(t testing.TB, encoders ...httpext.CompressionType) http.Handler`, where you would stack the compressors in the order provided.

Nevermind if it's too much work, just a thought.

Reply:
I thought about this, too. But the thing is that it's just a test. If we want to merge them, then we nearly re-introduce a function with the same mechanism that `compressBody` does. That's why I didn't merge them.